app/src/main/java/org/simple/clinic/summary/teleconsultation/TeleconsultationMessageBuilder.kt

```diff
@@ -34,37 +34,36 @@ class TeleconsultationMessageBuilder @Inject constructor(
         .appendln("https://app.simple.org/patient/${patientTeleconsultationInfo.patientUuid}")
         .appendln("")
 
-    message.appendln(resources.getString(R.string.patientsummary_contact_doctor_diagnosis))
+    diagnosis(patientTeleconsultationInfo, message)
+    bloodPressures(patientTeleconsultationInfo, message)
+    bloodSugars(patientTeleconsultationInfo, message)
+    prescriptions(patientTeleconsultationInfo, message)
```

Comment:
The names of these methods are not too descriptive since they do not expose the detail that the `message` parameter is going to be modified by them. Can we rename the methods so this detail is exposed?

Ex: `addDiagnosisSectionToMessage(patientTeleconsultationInfo, message)`.

Reply:
Done